### ISSUE TITLE
About the project

### ISSUE THREAD

**Issue by StefanPeev** [ssagano/Aileron#5]
@ssagano @davelab6 I want to ask about the project: is there an idea for development of the family Aileron or the project is finished?

**Comment by StefanPeev** [comment]
@davelab6 Dear Dave, there was a conversation before between you and @adrientetar (if I am not in mistake) for including Aileron in Google Fonts. Now the Aileron's license is changed to SIL and I have the written permission of Sora Sagano to add Cyrillic Script to Aileron. I have already sent a PR with Cyrillic Script for Aileron UltraLight which Sora Sagano merged in its repository. So I'm preparing the other styles of Aileron and think that it is a good idea Aileron to be part of Google Fonts. It's my propose to you.
Best regards,
Stefan

**Comment by davelab6** [comment]
Awesome!! :)

**Comment by StefanPeev** [comment]
@davelab6 Does it mean "Yes!", Dave :)

**Comment by davelab6** [comment]
Yes - I am keen to include this family, but there is a queue to getting
things pushed

**Comment by StefanPeev** [comment]
@davelab6 Oh, I've a lot of work to do before Aileron to start speaking Cyrillic, so there is time. But let we be on a queue – even if we are at the end :)
